src/main/java/com/conveyal/gtfs/loader/Table.java

```diff
@@ -172,7 +173,7 @@ public Table (String name, Class<? extends Entity> entityClass, Requirement requ
     public static final Table FEED_INFO = new Table("feed_info", FeedInfo.class, OPTIONAL,
         new StringField("feed_publisher_name", REQUIRED),
         // feed_id is not the first field because that would label it as the key field, which we do not want because the
-        // key field cannot be optional. feed_id is not part of the GTFS spec, but is required by OTP.
+        // key field cannot be optional. feed_id is not part of the GTFS spec, but is useful to OTP.
```

Comment:
This language change seems well-meaning, but doesn't provide any additional info and just makes things vague.
```suggestion
        // key field cannot be optional. feed_id is not part of the GTFS spec, but is required by OTP to associate static GTFS with GTFS-rt feeds.
```